arangod/V8Server/v8-collection.cpp

```diff
@@ -962,7 +962,7 @@ static void InsertVocbaseCol(TRI_vocbase_col_t* col, uint32_t argOffset,
   TRI_GET_GLOBALS();
 
   if (argLength < 1 || argLength > 2) {
-    TRI_V8_THROW_EXCEPTION_USAGE("insert(<data>, [<waitForSync>])");
+    TRI_V8_THROW_EXCEPTION_USAGE("save(<data>, [<waitForSync>])");
```

Comment:
This is a consistency change: the methods "save" and "insert" are synonymous but the error messages used the names at random. As "save" seems to be the preferred name at this point, I decided to be bold and adjust the messages accordingly.